### ISSUE TITLE
`--source-dir` folder name with whitespace causes failure to run `ash` tool

### ISSUE THREAD

**Issue by climbertjh2** [awslabs/automated-security-helper#58]
naming a `--source-dir` folder path something that contains a whitespace character, e.g. `my scan` vs. `my-scan` causes `ash` to fail to run.

Re-create:

```
mkdir "my scan"
cd "my scan"
git clone <repo to scan>
cd <repo to scan>
ash --source-dir .
```

**Comment by awsntheule** [comment]
Hey @climbertjh2, this should be fixed with PR #64. Please clone the latest version and try again when you get a chance.

**Comment by climbertjh2** [comment]
It appears to work - thanks for updating.  Closing this issue.
